operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -0,0 +1,173 @@
+# Create a Session Template to Boot Compute Nodes with Scalable Boot Projection Service (SBPS)
+
+When [compute nodes](../../glossary.md#compute-node-cn) are booted, the [Scalable Boot Projection Service (SBPS)](../../glossary.md#scalable-boot-projection-service-sbps)
+projects the root file system \(`rootfs`\) over the network to the compute nodes.
+
+
+This page covers the appropriate contents for a BOS session template in order to use SBPS.
+
+- [Boot set S3 parameters](#boot-set-s3-parameters)
+- [Boot set `rootfs_provider` parameters](#boot-set-rootfs_provider-parameters)
+  - [`<transport>`](#transport)
+  - [`<schema version>`](#schema-version)
+  - [`<IQN Domain>`](#iqn-domain)
+  - [`<DNS SRV record reference>`](#dns-service-srv-record-reference)
+  - [`<client discovery timeout in seconds>`](#client-discovery-timeout-in-seconds)
+  - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
+- [`root=` kernel parameter](#root-kernel-parameter)
+- [Example session template input file](#example-session-template-input-file)
+- [Creating a BOS session using the new template](#creating-a-bos-session-using-the-new-template)
+
+## Boot set S3 parameters
+
+The session template boot set contains several [Simple Storage Service (S3)](../../glossary.md#simple-storage-service-s3) parameters.
+These are listed below, along with the appropriate values to use.
+
+- `type`: Set to `s3`
+- `path`: Set to `s3://<BUCKET_NAME>/<KEY_NAME>`
+  - `<BUCKET_NAME>` is set to `boot-images`
+  - `<KEY_NAME>` is set to the image ID that the [Image Management Service (IMS)](../../glossary.md#image-management-service-ims) created when it generated the boot artifacts.
+- `etag`: Set to `<etag\>`
+
+## Boot set `rootfs_provider` parameters
+
+The Scalable Boot Projection Service (SBPS) is an optional provider for the `rootfs` on compute nodes.
+
+For SBPS, the `rootfs_provider_passthrough` boot set parameter is customized according to the following format:
+
+```text
+rootfs_provider_passthrough=<transport>:<schema version>:<IQN Domain>:<DNS SRV record reference>:<client discovery timeout in seconds>
+```
+
+The following values need to be set in the boot set of the session template in order to make SBPS the `rootfs` provider:
+
+- `"rootfs_provider":` Set to `"sbps"`
+- To use the Node Management Network (NMN) for content projection,
+  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-nmn._tcp.local:300"`
+- To use the High Speed Network (HSN) for content projection,
+  - `"rootfs_provide_passthrough"`: Set to `"sbps:v1:iqn.2023-06.csm.iscsi:_sbps-hsn._tcp.local:300"`
```

Comment:
This domain name will be based on the system domain name.

Reply:
I used an example using an HPE system domain name rather than the domain name from the IQN.